content/en/real_user_monitoring/error_tracking/mobile/reactnative.md

```diff
@@ -142,21 +158,23 @@ const crashApp = () => {
 };
 ```
 
-## Alternatives to `datadog-react-native-wizard`
+## Additional configuration options
+
+### Alternatives to `datadog-react-native-wizard` for symbolication
 
 If using `datadog-react-native-wizard` did not succeed or if you don't want to upload your symbolication files automatically on each release, follow the next steps to symbolicate crash reports.
 
-### Upload JavaScript source maps on iOS builds
+#### Upload JavaScript source maps on iOS builds
 
-You need to install `@datadog/datadog-ci` as a dev dependency to your project:
+First, you need to install `@datadog/datadog-ci` as a dev dependency to your project:
 
 ```bash
 yarn add -D @datadog/datadog-ci
 # or
 npm install --save-dev @datadog/datadog-ci
 ```
 
-#### Automatically on each release build (React Native >= 0.69)
+{{% collapse-content title="Automatically on each release build (React Native >= 0.69)" level="h5" %}}
```

Comment:
Oooooh, collapsible config! It's so cool!!!

Reply:
Ya it's @brett0000FF's creation. It is very cool and improves readability for pages like this!